client.go

```diff
@@ -95,6 +95,11 @@ func handleResp(req *http.Request) (respBodyObj ResponseBody, err error) {
 		return
 	}
 
+	if resp.StatusCode >= 400 && resp.StatusCode <= 500 {
+		log.Error("Server returned 'Unauthorized access' code ", resp.StatusCode)
```

Comment:
Note, this could be replaced with the following to supply the actual 4xx error to the user with a string like `Server returned 4xx error code: 401 Unauthorized`:

```suggestion
		log.Error("Server returned 4xx error code ", resp.Status)
```